CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+# 0.15.0 (wip)
+
+## Release
+
+- nns `v0.15.0`
+- sns `v0.0.13`
+- cmc `v0.0.9`
+- ledger `v0.0.6`
+- ckBTC `v0.0.2`
+- utils `v0.0.13`
+-
```

Comment:
Probably a redundant line.

Reply:
You mean the empty `-`? Yes, literally just fixed it